src/main/java/uk/gov/hmcts/reform/userprofileapi/controller/advice/ErrorConstants.java

```diff
@@ -14,9 +14,13 @@
 
     DATA_INTEGRITY_VIOLATION("6 : attempt to insert or update data resulted in violation of an integrity constraint"),
 
-    ILLEGAL_ARGUMENT("7 : method has been passed an illegal or inappropriate argument"),
+    USER_ALREADY_ACTIVE("7 : Resend failed because the user is Active. Wait for some time for the system to refresh."),

```

Comment:
one hour in the message should be a dynamic parameter

Reply:
This message is more  related to Sync timings. It always will be always 1 hour since sync is configured with 1 hour refresh time. So think this is ok. I have changed error message of too many requests within too many request.

Reply:
It is not about following what is happening in sync. .. i will be surprised if it is hardcoded in sync and not picked from the application yaml to run the scheduler. The review comment is more to follow the best practices around error messages. Any dynamic parameter in the error message should not be hardcoded and replaced dynamically in message so that it is easy to maintain in future